README.md

```diff
@@ -1,10 +1,28 @@
+[![GitHub issues open](https://img.shields.io/github/issues/inausoft/netCLI.svg?style=flat-square)]()
+[![NuGet version (inausoft.netCLI)](https://img.shields.io/nuget/v/inausoft.netCLI.svg?style=flat-square)](https://www.nuget.org/packages/inausoft.netCLI/)
+
+
 # netCLI
```

Comment:
There is no example that shows how to actually use this library. There are instructions how to write some code, but how to invoke the command from CLI?

Reply:
Please elaborate. What would you expect there to be?

Reply:
I'd expect something like "Usage" section which shows how to actually use this, eg:
```
> move --pathFrom foo --pathTo bar
```

Reply:
Agreed.